docs/requirements.txt

```diff
@@ -0,0 +1,2 @@
+Jinja2==2.7.3
+Sphinx==1.2.3
```

Comment:
It's been a long time, but what is the "Jinja2" dependency needed for? Won't Sphinx pull in some good version by itself?